zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -115,31 +115,6 @@ zipkin:
     mem:
       # Maximum number of spans to keep in memory.  When exceeded, oldest traces (and their spans) will be purged.
       max-spans: ${MEM_MAX_SPANS:500000}
-    cassandra:
```

Comment:
I noticed this was still in our yaml, though being dead for years, so I removed it.